cmd/mayactl/app/command/volume_info.go

```diff
@@ -160,17 +170,25 @@ func (c *CmdVolumeOptions) DisplayVolumeInfo(v *VolumeInfo, collection client.Re
 		portalInfo    PortalInfo
 	)
 	const (
-		replicaTemplate = `
-
+		jivaReplicaTemplate = `
 Replica Details :
-----------------
+-----------------
 {{ printf "NAME\t ACCESSMODE\t STATUS\t IP\t NODE" }}
 {{ printf "-----\t -----------\t -------\t ---\t -----" }} {{range $key, $value := .}}
 {{ printf "%s\t" $value.Name }} {{ printf "%s\t" $value.AccessMode }} {{ printf "%s\t" $value.Status }} {{ printf "%s\t" $value.IP }} {{ $value.NodeName }} {{end}}
 `
+
+		cstorReplicaTemplate = `
+Replica Details :
+-----------------
+{{ printf "%s\t" "NAME"}} {{ printf "%s\t" "ACCESS MODE"}} {{ printf "%s\t" "STATUS"}} {{ printf "%s\t" "POOL NAME"}} {{ printf "%s\t" "POOL IP"}} {{ printf "%s\t" "NODE"}}
```

Comment:
Since there are different templates, it is probably better to remove the Access Mode and Pool IP till they are supported.